### ISSUE TITLE
Changes made on lightning components not being detected

### ISSUE THREAD

**Issue by mauriciosonza** [celador/ForceCode#114]
Hi guys,
Currently I am using [TFS for VSCODE](https://github.com/Microsoft/vsts-vscode) along side with ForceCode because when I made changes on a class (.cls) file for example, the TFS plugin reconizes the change so I can comit it to my VSTS GIT repository (doing the version management) and also deploying it to the server. 
That was working good, till I tried to made changes on my lightning components. For some reason, they are not recognized by ForceCode to deploy on server.

Here is how my TFS structure is running right now : 

![p1](https://user-images.githubusercontent.com/7718233/28807325-5842671c-764b-11e7-8e19-9c66656c30ef.PNG)

Ths structure were created using [Gearset ](https://app.gearset.com) to deploy code from my org to the repository. 

I made a test pulling the code using ForceCode and I saw that he created another folder named 'src' and inside that lightnings are created inside another 'aura' folder. So I made a change on that component 'TFSComp' and it worked! (Change recognized and deployed on the server)

![p2](https://user-images.githubusercontent.com/7718233/28807565-92066b00-764c-11e7-9e9a-6a091ee6f90b.png)

So basically : 

Changes that I made on classes inside the 'classes' folder are working perfectly.
Changes that I made on lightning components inside 'src\aura' folder are being reconized.
Changes that I made on lightning components inside 'aura' folder are not being recognized, and that's a problem, since the folder and files are created dinamically and not by me, so I need that the changes made here were recognized as well to guarantee the code version management and so on.

This folder-structure-name thing is related on how ForceCode track changes or am I missing something?

**Comment by celador** [comment]
A typical Salesforce project has a src folder in it's root, which holds all the metadata and files for the Salesforce project.  If your code generator is creating stuff in the base folder you're working in, I'd recommend renaming that folder to `src` and opening the folder above that in VSCode, so your file tree will have a top level `src` folder.

Alternatively, review the configuration section of the Readme.  In there, it details the `src` configuration option.  It might be as simple as setting this option to `.` to refer to the current folder... who knows.  Please give it a shot and report back.
